book.toml

```diff
@@ -11,4 +11,19 @@ edition = "2021"
 
 [output.html]
 no-section-label = true
-git-repository-url = "https://github.com/time-rs/time"
+edit-url-template = "https://github.com/time-rs/book/edit/main/{path}"
+git-repository-url = "https://github.com/time-rs/book/tree/main/"
```

Comment:
It shouldn't be necessary to specify the branch.

Reply:
`git-repository-url` can just be https://github.com/time-rs/book, but the edit url seems to need the branch, I get a 404 otherwise.